cypress/integration/AdvancedSearchPage/AdvancedSearchForm/index.js

```diff
@@ -0,0 +1,33 @@
+/// <reference types="Cypress" />
+import { Then, When, And } from 'cypress-cucumber-preprocessor/steps';
+import { fieldMap } from '../../../utils/ctsFields.js';
+
+And('the text {string} appears below the title', (introtext) => {
+	cy.get('.search-page__header > p').should('contain', introtext);
+});
+
+And('Search tip icon is displayed and text {string} appears', (tiptext) => {
+	cy.get('.cts-search-tip__icon').should('be.visible');
+	cy.get('.cts-search-tip__body').should('contain', tiptext);
+});
+
+And(
+	'sticky block is displayed with {string} and {string} buttons',
+	(findbtn, clearbtn) => {
+		cy.get('.sticky-block').should('be.visible');
+		cy.get('.faux-btn-submit').contains(findbtn).should('be.visible');
+		cy.get('.clear-form').contains(clearbtn).should('be.visible');
+	}
+);
+
+When('user scrolls to the {string} section', (section) => {
+	cy.get(`input[aria-label="${section}"]`).scrollIntoView();
+});
+
+Then('sticky block becomes fixed block', () => {
+	cy.get('.sticky-block__anchor').should('be.visible');
```

Comment:
so this locator indicates an element, that is always visible
What you need to assert is that this div[class="sticky-block__anchor --sticky"] does't exist